pom.xml

```diff
@@ -2045,40 +2005,6 @@ under the License.
         <enabled>true</enabled>
       </releases>
     </repository>
-
-    <!-- Disable some Spring repositories pulled in by dependencies - see
```

Comment:
Why removing disabled Spring repositories here?

Reply:
Why would you want to keep disabled repositories?

Reply:
Isn't the information provided in the given link still valid?

Reply:
It is, but disabling a repository is effectively the same as a no-op. It's dead code, and certainly, the repositories are no longer needed and have been removed from upstream references. If you're telling maven, "don't use this repository for anything", then it would make sense to remove "unused" code.

Reply:
There are dependencies that have such repos defined hence Syncope builds will report permission errors; we added such definitions to stop pulling from such repos.

Reply:
Could you point out what those dependencies are? Can we upgrade those? And I don't follow. If the repository is 100% disabled, how are those dependencies resolved?

Reply:
When you remove such disabled repo definitions, you will notice that some 3rd party dependencies will be attempted to be downloaded from Spring repositories, failing with permission errors.
Anyway, let's keep the repo removal, I am going to build from your branch to see if I can still see such errors.